KlingonAssistant/data/mem-02-ch.xml

```diff
@@ -704,7 +704,7 @@ A {chaw':n:nolink} shows that you are officially or legally allowed to do or get
       <column name="entry_name">chay'?</column>
       <column name="part_of_speech">excl</column>
       <column name="definition">How did this happen?  What's going on?</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">Was? Wie konnte das passieren?</column>
```

Comment:
I'm just writing out the things that need to be fixed for myself. I'll merge everything, then do one more commit to fix all the miscellaneous things like newlines.